frontend/src/utils/AuthContext.tsx

```diff
@@ -100,24 +100,29 @@ export const AuthProvider = ({ children }: AuthProviderProps) => {
     "/login",
     "/signup",
     "/password/forgot",
-    "/password/reset/*",
+    "/password/reset/",
   ];
 
   // Paths that can be accessed freely when not logged in
   const publicPaths = [
     "/login",
     "/signup",
     "/password/forgot",
-    "/password/reset/*",
+    "/password/reset",
```

Comment:
why remove asteriks? I thought the asteriks is supposed to match all the links that have /password/reset?

Reply:
We found out that the asteriks does nothing it just takes in the string as it is, so the asteriks will never happen unless the route is named '*'